packages/frontend/tailwind.config.js

```diff
@@ -61,9 +64,11 @@ module.exports = {
       red: {
         300: '#FA3A3A',
         500: '#C32806',
+        600: '#C7141433',
```

Comment:
when you specify bg-red-600 you should add bg-opacity-10 instead of adding alpha

Reply:
Just curious, why is it better?